mmdet/core/bbox/samplers/ohem_sampler.py

```diff
@@ -21,20 +21,21 @@ def __init__(self,
                  **kwargs):
         super(OHEMSampler, self).__init__(num, pos_fraction, neg_pos_ub,
                                           add_gt_as_proposals)
-        if not hasattr(context, 'num_stages'):
-            self.bbox_roi_extractor = context.bbox_roi_extractor
-            self.bbox_head = context.bbox_head
+        self.context = context
+        if not hasattr(self.context, 'num_stages'):
+            self.bbox_head = self.context.bbox_head
         else:
-            self.bbox_roi_extractor = context.bbox_roi_extractor[
-                context.current_stage]
-            self.bbox_head = context.bbox_head[context.current_stage]
+            self.bbox_head = self.context.bbox_head[self.context.current_stage]
```

Comment:
There is something wrong with the OHEM usage in cascade rcnn. May I ask if you would like to fix that? If not, you can add an assertion here and I will fix it soon.

Reply:
Yes, I'd like to fix that, but I only have time on weekends

Reply:
> There is something wrong with the OHEM usage in cascade rcnn. May I ask if you would like to fix that? If not, you can add an assertion here and I will fix it soon.

Yes, I'd like to fix that, but I only have time on weekends.

Reply:
No problem

Reply:
> No problem

I've fixed the OHEM usage problem in cascade rcnn

Reply:
> No problem

I fixed the OHEM usage problem in cascade rcnn，please have a check